user_data/strategies/Strategy002.py

```diff
@@ -167,8 +168,13 @@ def confirm_trade_entry(self, pair: str, order_type: str, amount: float, rate: f
         coin = pair.split("/")[0]
         brain = "Freq_" + self.__class__.__name__
         if IS_BACKTEST:
-            threading.Thread(target=back_tester, args=(current_time, coin, brain)).start()
+            print("Strategy002:current_time = "+str(current_time))
+            if current_time.year != CURRENT_YEAR or current_time.month != CURRENT_MONTH:
```

Comment:
Why do we need this logic?
It seems like a hack

If we fix the real cause through the script, we might not need this workaround

Reply:
Yes, that's what I meant yesterday. 

This is also a valid solution, but the logic is not clean:

Use the following:

if currenttime.year is BACKTEST_YEAR and currenttime.month is BACKTEST_MONTH_INDEX + 1:
 threading.Thread
else 
 print ("redundant data")
 pass

Reply:
Yes here is the message from yesterday:
"Ok, you can write a script to cut the redundant data out of the file"

Please write a script to cut out redundant data out of the downloaded file.

What you did here is a hack not a fix

Reply:
please name the script: freq_data_cleaner.py

Reply:
Ok I understand thanks